src/views/Dashboard/Lab/Services/Add.vue

```diff
@@ -452,47 +463,48 @@ export default {
       if (!this.$refs.addServiceForm.validate()) {
         return
       }
-      this.isLoading = true
-      await createService(
-        this.api,
-        this.pair,
-        {
-          name: this.name,
-          pricesByCurrency: [
-            {
-              currency: this.currencyType,
-              priceComponents: [
-                {
-                  component: "component_1",
-                  value: this.web3.utils.toWei(String(this.price), 'ether')
-                }
-              ],
-              additionalPrices: [
-                {
-                  component: "qc_component",
-                  value: this.web3.utils.toWei(String(this.qcPrice), 'ether')
-                }
-              ],
+      try {
+        this.isLoading = true
+        await createService(
+          this.api,
+          this.wallet,
+          {
+            name: this.name,
+            pricesByCurrency: [
+              {
+                currency: this.currencyType,
+                totalPrice: await toEther(this.price + this.qcPrice),
+                priceComponents: [
+                  {
+                    component: "test_price",
```

Comment:
should re-check if this is test_price or testing_price?

Reply:
Hi mbak @hilyds, I've tried to double-check, mostly on the backend they use `testing_price`, and for `test_price` I can't find any. Is it using `testing_price`? please check my cross-check video below 👇🏼 

https://www.awesomescreenshot.com/video/6158148?key=89324edf962cc84d6b403c7b79ecb641